kit/ForKit.cpp

```diff
@@ -423,11 +430,11 @@ static int createLibreOfficeKit(const std::string& childRoot,
             LOG_INF("Forked kit [" << pid << ']');
             childJails[pid] = childRoot + jailId;
         }
-
-#ifndef KIT_IN_PROCESS
-        UnitKit::get().launchedKit(pid);
-#endif
+        if (!Util::isKitInProcess()) {
```

Comment:
Please do not use javascript formatting in C++ code. Apply clang-format if it's too much work.